SCOPES.md

```diff
@@ -0,0 +1,57 @@
+# Prefixes
+Supported scope prefixes:
+* `project:releng:beetmover`
+  * Used for Firefox/Fennec/Devedition style beetmover tasks
+
+* `project:comm:thunderbird:releng:beetmover`
+  * Used for Thunderbird style beetmover tasks
+
+
+# Scopes
+* `{scope_prefix}:action:push-to-candidates`
+  * Tells beetmoverscript to copy to candidates directory in S3 the corresponding release artifacts
+  * **Conflicts**: with any other `{scope_prefix}:action:*`
+  * **Branch Restrictions**: `all-candidates-tasks`
+
+* `{scope_prefix}:action:push-to-release`
+  * Tells beetmoverscript to move the artifacts from candidates directory to releases directory for the corresponding release
+  * **Conflicts**: with any other `{scope_prefix}:action:*`
+  * **Branch Restrictions**: `all-push-tasks`
```

Comment:
Restrictions here too (I'm going to omit commenting on other restriction lines in this review pass)